convert_zdf.py

```diff
@@ -0,0 +1,168 @@
+"""Convert ZDF point cloud to the available formats.
+
+Available formats:
+    .ply - Polygon File Format
+    .csv,.txt - [X, Y, Z, r, g, b, Contrast]
+    .png,.jpg,.bmp,.tiff - 2D RGB image
+"""
+
+import argparse
+import os
+import numpy as np
+import cv2
+import zivid
+
+
+def _options():
+    parser = argparse.ArgumentParser(
+        description="Convert from ZDF to your preferred format",
+        epilog="Example: $ python -u convert_zdf.py --csv --png Zivid3D.zdf Zivid3D_1.zdf",
+    )
+
+    parser.add_argument(
+        "files", nargs="*", help="List of files to convert", default="Zivid3D.zdf"
+    )
+    parser.add_argument("--ply", action="store_true", help="Convert to PLY")
+    parser.add_argument("--csv", action="store_true", help="Convert to CSV")
+    parser.add_argument("--txt", action="store_true", help="Convert to text")
+    parser.add_argument(
```

Comment:
Are you supposed to add multiple arguments here? --txt --ply --csv ....

Reply:
I want each to be a Boolean. It could have been a list of course, but the thought is that only a few would be requested at a time

Reply:
I mean: are these mutually exclusive?

Reply:
And if they are made mutally exclusive, also make them required=True

Reply:
no, they are not mutually exclusive. I did consider just adding a list of optional 2d formats, but didn't want to stray from "--format" syntax. If you believe it's better I could change it to (example) "--2d png bmp jpg"

Reply:
Made it mutually exclusive